CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 ## current master
 
-* update to oshdb version 0.5.10 (including an Ignite update to 2.9.0)
+### Prominent Changes 
+
+* several improvements to the docs, e.g. ([#49](https://github.com/GIScience/ohsome-api/pull/49)) and partially in ([#34](https://github.com/GIScience/ohsome-api/pull/34)) 
+* upgrade to ohsome-filter 1.3 ([#62](https://github.com/GIScience/ohsome-api/pull/62))
+
+### New Features
+
+* adding new endpoint /contributions ([#34](https://github.com/GIScience/ohsome-api/pull/34))
+
+### Bug Fixes
+
+* fixing not-encoded pipe sign in GET requests ([#60](https://github.com/GIScience/ohsome-api/pull/60)) ([#61](https://github.com/GIScience/ohsome-api/pull/61))
+
+### Other Changes
+
+* update to oshdb version 0.5.10 (including an Ignite update to 2.9.0) ([#67](https://github.com/GIScience/ohsome-api/pull/67)) 
+* diverse code refactoring improvements, e.g. ([#46](https://github.com/GIScience/ohsome-api/pull/46)), ([#47](https://github.com/GIScience/ohsome-api/pull/47)), ([#51](https://github.com/GIScience/ohsome-api/pull/51)), ([#57](https://github.com/GIScience/ohsome-api/pull/57))
+* include test directory in checkstyle ([#69](https://github.com/GIScience/ohsome-api/pull/69))
 
```

Comment:
```suggestion

## 1.1.1

### Bug Fixes

* allowing not encoded pipe sign in GET requests ([#60](https://github.com/GIScience/ohsome-api/pull/60))
```

Reply:
Also patch release notes should go into the main branch!

Reply:
Really? But then if we have at some point different versions running and are fixing different bugs in different versions, it could get a bit out of order in the changelog of the main branch.

Reply:
hm, I'd say it's quite simple: the changelog in the main branch just has to hold all changes of all versions ever released, in chronological order.

> And your initial comment is missleading, 

sorry, I didn't see that the bullet point at first. that's why I added these secondary review comments.

What I think is important is that the whole existence of version `1.1.1` was (and is still) missing in the changelog. the changelog should always be complete.

PS: You might choose to list a bugfix twice in the changelog if it was backported, so feel free to ignore the respective suggestion [above](https://github.com/GIScience/ohsome-api/pull/71#discussion_r518901890).

Reply:
What I mean is that you could then get something like the following:

1.1.1 bug fix
1.2 release
2.0 release
1.2.1 bugfix
2.1 release
1.2.2 bugfix
etc

I just was not aware that it's fine to have it like that in the changelog then (in the order from top to bottom).

Reply:
why not?